src/LocalField/neq.jl

```diff
@@ -0,0 +1,216 @@
+#export: degree_relative, random_elem, one_root, norm_equation  
+
+
+degree_relative(L::Hecke.LocalField,K::Hecke.LocalField) = Int(absolute_degree(L)//absolute_degree(K)
+)
+
+ degree_relative(L::Hecke.RelFinField,K:: Hecke.RelFinField)  = Int(absolute_degree(L)//absolute_degree(K))
+degree_relative(L::Hecke.RelFinField,K::FqNmodFiniteField)  = Int(absolute_degree(L)//absolute_degree(K))
+
+degree_relative(L::FqNmodFiniteField, K::Hecke.Nemo.GaloisField)  = Int(absolute_degree(L)//absolute_degree(K))
+
+
+##############################################
+#random element with small coefficients
+##############################################
+
+function random_elem(L::Hecke.LocalField)
+     b = basis(L)
+     n = degree(L)
+     r = [rand(1:5*n) for i in 1:n]
+return sum( [ r[i]*b[i] for i in 1:n])
```

Comment:
This code is unevenly indented, fix that? I.e. this line is not indented at all, similarly elsewhere. Also, the lines before seem to use 5 spaces for indentation, which is highly unusual (I've seen 2, 3 and 4 used in various Oscar projects and elsewhere, but I think never ever 5, in the past 20 years :-)

Reply:
Thanks for your nice suggestion.s I will change all  accordingly